gender_novels/novel.py

```diff
@@ -137,17 +139,72 @@ def _load_novel_text(self):
             err += "at the expected location ({file_path})."
             raise FileNotFoundError(err)
 
-        # Extract Project Gutenberg Boilerplate
-        if text.find('*** START OF THIS PROJECT GUTENBERG EBOOK') > -1:
-            end_intro_boilerplate = text.find(
-                '*** START OF THIS PROJECT GUTENBERG EBOOK')
-            # second set of *** indicates start
-            start_novel = text.find('***', end_intro_boilerplate + 5) + 3
-            end_novel = text.find('*** END OF THIS PROJECT GUTENBERG EBOOK')
-            text = text[start_novel:end_novel]
+        # These two functions will ensure that the text becomes only the novel's own text,
+        # removing the Project Gutenberg .
+        text = self.remove_boilerplate_text(text)
+        text = self.remove_table_of_contents(text)
+
+        return text
+
+
+    def remove_boilerplate_text(text):
```

Comment:
This function should include the existing Project Gutenberg boiler plate removal:
```
        if text.find('*** START OF THIS PROJECT GUTENBERG EBOOK') > -1:
            end_intro_boilerplate = text.find(
                '*** START OF THIS PROJECT GUTENBERG EBOOK')
            # second set of *** indicates start
            start_novel = text.find('***', end_intro_boilerplate + 5) + 3
            end_novel = text.find('*** END OF THIS PROJECT GUTENBERG EBOOK')
            text = text[start_novel:end_novel]
```

Reply:
I can put in the existing boiler plate remover but will comment it out and add a note, as there are certain books which actually do not have the standard project gutenberg boilerplate for which that function does not work (eg, james_highway.txt).

The gutenberg strip_headers() function does work on james_highway.txt however, so it is believed that strip_headers() works for all boilerplates, including nonstandard ones.